features/yield/FarmingCalculator.tsx

```diff
@@ -306,46 +359,51 @@ const FarmingCalculator = () => {
         )}
         <br></br>
         <br></br>
-        <Typography>
-          <strong>Tokens eligible for liquidity mining:</strong>{" "}
-          {timeUntilRoll
-            ? isRolled
-              ? rollTokenName
-              : `${tokenSymbol} + ${rollTokenName}`
-            : tokenSymbol}
-          <br></br>
-          {hoursRemainingToFarmingRoll && hoursRemainingToFarmingRoll > 0 && (
-            <>
-              <strong>Hours remaining until liquidity mining rolls:</strong>{" "}
-              {hoursRemainingToFarmingRoll.toFixed(2)}
-              <br></br>
-            </>
-          )}
-          <strong>Total liquidity eligible for mining rewards:</strong> $
-          {Number(poolLiquidity).toLocaleString()}
-          <br></br>
-          {rewardToken &&
-            rewardToken.map((rewardObj) => {
-              return (
-                rewardYieldAmounts[rewardObj.token] &&
-                rewardTokenPrices[rewardObj.token] && (
-                  <span key={rewardObj.token}>
-                    <br></br>
-                    <strong>
-                      Weekly {rewardObj.token} distributed to pool:
-                    </strong>
-                    {" " + rewardObj.count.toLocaleString()} ($
-                    {rewardYieldAmounts[rewardObj.token].toLocaleString()})
-                    <br></br>
-                    {`- 1 ${rewardObj.token} = $${rewardTokenPrices[
-                      rewardObj.token
-                    ].toLocaleString()}`}
-                  </span>
-                )
-              );
-            })}
-        </Typography>
-
+        {rewardToken.length > 0 ? (
```

Comment:
This moves the entire `Typography` block inside this conditional. If there are no reward tokens to display, simply display to user that rewards are not currently available for this token